modules/apps/headless/headless-admin-user/headless-admin-user-impl/src/main/java/com/liferay/headless/admin/user/internal/jaxrs/exception/mapper/UserAccountInvalidFirstNameExceptionMapper.java

```diff
@@ -0,0 +1,38 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.headless.admin.user.internal.jaxrs.exception.mapper;
+
+import com.liferay.portal.kernel.exception.ContactNameException;
+import com.liferay.portal.vulcan.jaxrs.exception.mapper.BaseExceptionMapper;
+import com.liferay.portal.vulcan.jaxrs.exception.mapper.Problem;
+
+import javax.ws.rs.core.Response;
+import javax.ws.rs.ext.ExceptionMapper;
+
+import org.osgi.service.component.annotations.Component;
+
+/**
+ * @author Michele Vigilante
+ */
+@Component(
+	property = {
+		"osgi.jaxrs.application.select=(osgi.jaxrs.name=Liferay.Headless.Admin.User)",
+		"osgi.jaxrs.extension=true",
+		"osgi.jaxrs.name=Liferay.Headless.Admin.User.UserAccountInvalidFirstNameExceptionMapper"
+	},
+	service = ExceptionMapper.class
+)
+public class UserAccountInvalidFirstNameExceptionMapper
+	extends BaseExceptionMapper<ContactNameException.MustHaveValidFirstName> {
+
+	@Override
+	protected Problem getProblem(
+		ContactNameException.MustHaveValidFirstName mustHaveValidFirstName) {
+
+		return new Problem(mustHaveValidFirstName);
```

Comment:
Let's check this because the new pattern of the mappers should be
```
return new Problem(
			Response.Status.BAD_REQUEST, "User account given name is invalid");
```
like the one we sent last time. 
`User account` because it's the user account
`given name` because in the yaml (so in the response of the web service) the firstName is called given name
`is invalid` because has an invalid value

@riccardo-alberti we need your help on this PR. Thx

Reply:
See if brian answer [here](https://github.com/brianchandotcom/liferay-portal/pull/152925#issuecomment-2265940146) about the new pattern